compose.prod.yaml

```diff
@@ -91,6 +91,8 @@ x-airflow-common:
     AIRFLOW_VAR_SUL_PUB_HOST: ${AIRFLOW_VAR_SUL_PUB_HOST}
     AIRFLOW_VAR_SUL_PUB_KEY: ${AIRFLOW_VAR_SUL_PUB_KEY}
     AIRFLOW_VAR_DATA_DIR: /opt/airflow/data
+    AIRFLOW_VAR_GOOGLE_CONNECTION: google_cloud_default
```

Comment:
this should be the same for all environments and will be setup manually in the airflow UI as described in the README